xmtp_db/src/encrypted_store/group.rs

```diff
@@ -813,6 +813,7 @@ impl<C: ConnectionExt> QueryGroup for DbConnection<C> {
                             .set(dsl::welcome_id.eq(group.welcome_id))
                             .execute(c)
                     })?;
+                    existing_group.welcome_id = group.welcome_id;
                 }
                 Ok(existing_group)
```

Comment:
Unrelated change - it looks like before this, stale data was being returned from this method